eolearn/io/raster_io.py

```diff
@@ -42,39 +42,43 @@ class BaseRasterIoTask(IOTask, metaclass=ABCMeta):
     def __init__(
         self,
         feature: Feature,
-        folder: str,
+        path: str = ...,  # type: ignore[assignment]
         *,
         filesystem: FS | None = None,
         image_dtype: np.dtype | type | None = None,
         no_data_value: float | None = None,
         create: bool = False,
         config: SHConfig | None = None,
+        folder: str | None = None,
     ):
         """
         :param feature: Feature which will be exported or imported
-        :param folder: A path to a main folder containing all image, potentially in its subfolders. If `filesystem`
-            parameter is defined, then `folder` should be a path relative to filesystem object. Otherwise, it should be
-            an absolute path.
+        :param path: A path to the image(s). If `filesystem` is defined, then `folder` should be given relative to the
+            filesystem object. Otherwise, it should be an absolute path.
```

Comment:
This is a bit confusing, since you mention `folder`, which seems like a reference to the parameter which has been deprecated.

```suggestion
        :param path: Path pointing to an image file or to a directory of image files. If `filesystem` is not defined, absolute paths should be provided.
```